core.ts

```diff
@@ -26,6 +26,9 @@ export interface Data {
   /** To configure a different template engine(s) to render a page */
   templateEngine?: string | string[];
 
+  /** Dependencies to watch for this page */
+  dependencies?: string[];
```

Comment:
Loaders can now return a `dependencies` array. When those dependencies changes we will re-evaluate this file.